default_ignore_datasets.yaml

```diff
@@ -0,0 +1,10 @@
+esmvaltool:
```

Comment:
@lewisjared Is this the approach you were looking for? Or would you prefer to have separate files per provider?